src/core/ComboBox/ComboBox.tsx

```diff
@@ -545,62 +522,73 @@ const ComboBoxInput = React.forwardRef(
           case 'ArrowDown': {
             event.preventDefault();
             if (!isOpen) {
-              return dispatch(['open']);
+              dispatch(['open']);
```

Comment:
I'd say, if you open, just focus first item too.

Reply:
there is no `return`, so it will perform the appropriate action after opening

Reply:
I dont think it fully performs the appropiate action now.
![image](https://user-images.githubusercontent.com/81580355/165279848-cff0ee4e-6eae-4d23-b333-5c21067fbd9f.png)

After selected value and opening again with keyboard, it does not focus selected value.